harper-core/src/linting/oxford_comma.rs

```diff
@@ -21,7 +21,7 @@ impl OxfordComma {
                 ))
                 .then_noun_phrase()
                 .then_whitespace()
-                .then(Box::new(WordSet::all(&["and", "or"])))
+                .then(Box::new(WordSet::all(&["and", "or", "nor"])))
```

Comment:
The description function around line 80 also need to be updated.

And I like the fact you will have to use an Oxford list to write it 😁